datafusion/core/tests/sql/timestamp.rs

```diff
@@ -1398,6 +1398,7 @@ async fn timestamp_sub_interval_days() -> Result<()> {
 }
 
 #[tokio::test]
+#[ignore] // https://github.com/apache/arrow-datafusion/issues/3327
```

Comment:
Should this test be ignored? Maybe this is a merge conflict -- I think @HaoYang670  has fixed this test in https://github.com/apache/arrow-datafusion/pull/3337 so it no longer needs to be ignored

Reply:
Thanks for catching that. Yes, this must have been some kind of merge conflict. I have unignored this.